src/android.ts

```diff
@@ -110,19 +110,31 @@ async function getLldbServer(abi: string): Promise<string> {
         let platform = os.platform();
 
         let llvmHostName = (platform === "win32") ? "windows-x86_64" : (platform === "darwin") ? "darwin-x86_64" : "linux-x86_64";
-        let llvmArch = abi.startsWith("armeabi") ? "arm" : (abi === "arm64-v8a") ? "aarch64": (abi === "x86") ? "i386" : "x86_64";
+        let llvmArch = abi.startsWith("armeabi") ? "arm" : (abi === "arm64-v8a") ? "aarch64" : (abi === "x86") ? "i386" : "x86_64";
 
         let llvmToolchainDir = path.normalize(`${androidPaths.requireNdkRoot()}/toolchains/llvm/prebuilt/${llvmHostName}`);
 
         let lldvPackageVersion = (await fsPromises.readFile(path.join(llvmToolchainDir, "AndroidVersion.txt"), "utf8")).split("\n")[0];
 
-        let lldbServerPath = path.normalize(`${llvmToolchainDir}/lib64/clang/${lldvPackageVersion}/lib/linux/${llvmArch}/lldb-server`);
-
-        await fsPromises.access(lldbServerPath, fs.constants.R_OK);
+        const toolchainLibsPath = [
+            `lib64/clang/${lldvPackageVersion}`, // Old path for NDK <= 25
+            `lib/clang/${lldvPackageVersion}`,
+            `lib/clang/${lldvPackageVersion.substring(0, lldvPackageVersion.indexOf("."))}`
```

Comment:
Should we change it to `${lldvPackageVersion.split(".")[0]}` to avoid empty string in case the version does not have a `.` in it? Sorry, didn't notice this earlier. :/